src/components/Navbar.tsx

```diff
@@ -0,0 +1,230 @@
+        width: '2px',
+        background: 'yellow.5',
+        position: 'absolute',
+        left: 0
+      }}
+    >
+      <HStack spacing={2}>{children}</HStack>
+    </MenuItem>
+  );
+};
+
+const DeskLiItem = ({ children }: { children: ReactNode }) => {
+  return (
+    <HStack
+      pos='relative'
+      _hover={{
+        color: 'yellow.5',
+        transition: 'width 0.3s',
+        _after: { width: '100%' }
+      }}
+      _after={{
+        content: '""',
+        display: 'block',
+        width: 0,
+        height: '2px',
+        background: 'yellow.5',
+        transition: 'width .3s',
+        position: 'absolute',
+        bottom: 0,
+        left: 0
+      }}
+    >
+      <Text>{children}</Text>
+    </HStack>
+  );
+};
+
+const Navbar = () => {
+  const [isLogin, setIsLogin] = useState<boolean>(false);
+
+  // contoh
+  const handleLogin = () => {
+    setIsLogin(!isLogin);
+  };
+
+  return (
+    <Center>
+      <Flex
+        mt='1rem'
+        px={{ base: '3rem', lg: '5rem' }}
+        py='8px'
+        width='90%'
+        bgImage='/images/nav-bg.png'
+        bgColor='black'
+        backgroundSize='cover'
+        backgroundPosition='center'
+        backgroundRepeat='no-repeat'
+        justifyContent='space-between'
+        alignItems='center'
+        borderRadius='50'
+        boxShadow='0px 0px 20px 0px #FFFC83;'
+        overflow='hidden'
+        pos='fixed'
+        top='0'
+        color='white'
+        zIndex='100'
+        fontSize='14px'
+        fontWeight='semibold'
+      >
+        <Box pos='absolute' width='100%' left='0' top='0'>
+          <Image src='/images/nav-ekor.png' draggable='false' loading='lazy' />
+        </Box>
+        <Box zIndex='1000'>
+          <Image
+            src='/images/nav-logo.svg'
+            height={{ base: '38px', lg: '57px' }}
+            draggable='false'
+            loading='lazy'
+          />
+        </Box>
+        <UnorderedList
+          listStyleType='none'
+          display={{ base: 'none', lg: 'block' }}
+        >
+          <HStack spacing={{ lg: '27px', xl: '45px' }}>
+            <Link href='/tes'>
+              <DeskLiItem>About Us</DeskLiItem>
+            </Link>
+            <Link href='/'>
+              <DeskLiItem>Merchandise</DeskLiItem>
+            </Link>
+            <Link href='/'>
+              <DeskLiItem>Interactive Map</DeskLiItem>
+            </Link>
+            <Link href='/'>
+              <DeskLiItem>Blog</DeskLiItem>
+            </Link>
+            <Box onClick={handleLogin}>
+              {isLogin ? (
+                <Link href='/'>
+                  <Button variant='solid'>Space Log</Button>
+                </Link>
+              ) : (
+                <Link href='/'>
+                  <Button variant='outline'>Login</Button>
+                </Link>
+              )}
+            </Box>
+          </HStack>
+        </UnorderedList>
+        <Box display={{ base: 'block', lg: '' }} >
```

Comment:
```suggestion
        <Box display={{ base: 'block', lg: 'none' }} >
```

di test jadi gini soalnya
![image](https://github.com/KATITB2023/oskm-info/assets/65522975/04277eb6-45b3-4bbd-b5e0-924cd09792b1)

Reply:
oiya lupa blm dibuat none lagi